packages/odyssey-react/src/components/Box/Box.theme.ts

```diff
@@ -13,16 +13,9 @@
 import type { ThemeReducer } from "@okta/odyssey-react-theme";
 
 export const theme: ThemeReducer = (theme) => ({
-  // REMOVE THIS
   /* eslint-disable @okta/odyssey/no-invalid-theme-properties */
-
-  // Font
-  FontFamilyBase: theme.FontFamilyBase,
-  FontWeightNormal: theme.FontWeightNormal,
-  FontStyleNormal: theme.FontStyleNormal,
-  FontSize: theme.FontSizeBody,
-  FontLineHeightBase: theme.FontLineHeightBase,
-  ColorTextBody: theme.ColorTextBody,
+  // Width/Height
+  MaxLineLength: theme.FontLineLengthMax,
```

Comment:
this has been named `MaxWidth` elsewhere which allows it to work as a css declaration suffix and doesn't require eslint escaping the rule

Reply:
The uses within `Box` apply more broadly than `MaxWidth` - for example when the `max-width` is calculated by combining the maximum line length with some spacing units.

Reply:
@edburyenegren-okta That calc processing is happening elsewhere outside of `Box` too! I would favor the CSS Declaration style for now. If the name sucks or is unclear for folks we will change it.

In either case  (`MaxLineLength` or `MaxWidth) the custom property is going to be calc/transformed further such that the user provided value won't explicitly be used in the DOM!

Reply:
`MaxLineLength` and `max-width` represent different concerns, which is why `MaxLineLength` is usually part of a calc.

`MaxLineLength` refers to the amount of text characters per line. Our current value fixes it to ~45 characters with our preferred font family. We may update this to ~70 characters (the upper reasonable limit) in the future without changing any of the `calc()` expressions that determine `max-width`.

`max-width` refers to the allowable line length + padding + border (as long as we're using `border-box` model). It will usually be noticeably larger than `MaxLineLength`.

The difference between these two values is important for Designers and Developers understanding why widths are rendered the way they are. At the simplest, if we name this `MaxWidth` it's likely that the value a developer inserts will never match the rendered output of `max-width`.

Reply:
For clarity, I'm not strongly opinionated about the way we handle maximum character count and max-width. We could find a way to allow `MaxWidth` and `MaxLineLength` to be synonymous via some refactors.

My concern is more that `MaxWidth` doesn't accurately represent the API as it stands today.

Reply:
I think that's valid, let's keep it MaxLineLength for now